lib/src/default_index_store.rs

```diff
@@ -1802,7 +1805,7 @@ impl Index for ReadonlyIndexImpl {
         CompositeIndex(self).heads(candidates)
     }
 
-    fn topo_order(&self, input: &mut dyn Iterator<Item = &CommitId>) -> Vec<IndexEntry> {
+    fn topo_order(&self, input: &mut dyn Iterator<Item = &CommitId>) -> Vec<CommitId> {
```

Comment:
I don't know if we need this function at all. It seems that one can simply create a revset and iterate over it in reverse order and it'll be about as fast. But it doesn't do any harm right now, so I'm just leaving it here.